tests/docker-compose.yml

```diff
@@ -2,20 +2,40 @@ version: '3.1'
 
 services:
 
-  db:
-    image: postgres:9.0
+  db90:
+    image: postgres:9.0.22
     restart: always
     environment:
       POSTGRES_USER: postgres
       POSTGRES_PASSWORD: example
       POSTGRES_DB: demo
 
+  db91:
```

Comment:
I would suggest a more descripitve name for these. What about, for example, `postgres-9-0`?

Reply:
👍